packages/@aws-cdk/aws-docdb/lib/cluster.ts

```diff
@@ -470,6 +491,21 @@ export class DatabaseCluster extends DatabaseClusterBase {
     });
   }
 
+  /**
+   * Sets up CloudWatch log retention if configured.
+   */
+  private setLogRetention(cluster: DatabaseCluster, props: DatabaseClusterProps, cloudwatchLogsExports: string[]) {
+    if (cloudwatchLogsExports && props.cloudWatchLogsRetention) {
```

Comment:
No need for the first condition - `cloudwatchLogsExports` is never `undefined`, and an empty array is truthy in JavaScript:

```suggestion
    if (props.cloudWatchLogsRetention) {
```